styles/planet-minecraft/catppuccin.user.css

```diff
@@ -151,47 +178,6 @@
       background: @surface0 !important;
     }
 
-    /* because the icons on the display mode buttons are white, we cant have white base here, so we improvise for latte */
-    & when (@lookup = latte) {
-      #display_modes a {
-        background: @text !important;
-        box-shadow: none !important;
-      }
-
-      /* fix nav menu dropdown icons */
-      .navbar_icon .dropdown {
-        --svozk: var(--s) -256px -326px !important;
-        background: var(--s) -256px -326px !important;
-      }
```

Comment:
@nyameliaaaa do you remember where this stuff is? I'm a bit confused by the comment above and why this is necessary to be only for light mode. There might be a better solution or maybe we don't need this at all.

Reply:
i think you'll have to contact her on discord since shes rarely on catppuccin's github